client/jetpack-connect/signup.js

```diff
@@ -59,6 +70,7 @@ export class JetpackSignup extends Component {
 		isCreatingAccount: false,
 		newUsername: null,
 		bearerToken: null,
+		showWooDnaLoginForm: true,
```

Comment:
This param is being used to either show the log in form or the sign up for, but specifically only if it's a "woo dna" flow. Have you considered having it store the type of form we want to display instead of a boolean? For example either "signup" or "login", etc?

Reply:
I've changed it in https://github.com/Automattic/wp-calypso/pull/41798/commits/433900c08ffa2a00f80addc7b9be9fb3921470c0, I think it reads a bit better now :)